portfolio/src/main/webapp/sheetsApi.js

```diff
@@ -0,0 +1,133 @@
+
+// Client ID and API key from the Developer Console
+var CLIENT_ID =
+    '499747085593-hvi6n4kdrbbfvcuo1c9a9tu9oaf62cr2.apps.googleusercontent.com';
+var API_KEY = 'AIzaSyAcS8-F743Xy2ZTSYxxZIB0nY5kDQQ8i7E';
+
+// Array of API discovery doc URLs for APIs used by the quickstart
+var DISCOVERY_DOCS =
+    ['https://sheets.googleapis.com/$discovery/rest?version=v4'];
+
+// Authorization scopes required by the API; multiple scopes can be
+// included, separated by spaces.
+var SCOPES = 'https://www.googleapis.com/auth/spreadsheets';
```

Comment:
I personally added what the scope means in the variable name in my implementation, so maybe something like SCOPE_READ_WRITE would be more clear in indicating what this scope lets the user to do.

Reply:
Done.